corehq/apps/users/views/mobile/users.py

```diff
@@ -1184,6 +1184,7 @@ class FilteredUserDownload(BaseManageCommCareUserView):
     @method_decorator(require_can_edit_commcare_users)
     def get(self, request, domain, *args, **kwargs):
         form = CommCareUserFilterForm(request.GET, domain=domain, couch_user=request.couch_user)
+        form.empty_permitted = True
```

Comment:
nit: can you add a comment on why its needed. I think it will leave people confused.